src/mgr/Action.cc

```diff
@@ -120,3 +119,20 @@ Mgr::Action::fillEntry(StoreEntry* entry, bool writeHttpHeader)
         entry->complete();
 }
 
+void
+Mgr::openKidSection(StoreEntry *entry, const bool is_yaml)
+{
+    if (is_yaml)
+        storeAppendPrintf(entry, "---\nkid: %d\n", KidIdentifier);
+    else
+        storeAppendPrintf(entry, "by kid %d {\n", KidIdentifier);
```

Comment:
Is this change in framing markers in legacy reports intentional? If it is accidental, please undo here and in closeKidSection():

```suggestion
        storeAppendPrintf(entry, "by kid%d {\n", KidIdentifier);
```

Otherwise, please discuss why this PR should change that legacy (i.e. non-YAML) format.

Reply:
This had not been caught before - it's been like this for 6 weeks. Reverting